tests/unit/core/client/test_base.js

```diff
@@ -0,0 +1,51 @@
+import createClient from 'core/client/base';
+
+describe(__filename, () => {
+  describe('createClient()', () => {
+    const fakeCreateStore = () => {
+      return {
+        sagaMiddleware: null,
+        store: null,
+      };
+    };
+
+    const fakeFastClick = {
```

Comment:
I'd rather see this initialized before every test otherwise the call history of the stub *might* leak across tests. It might not because of [sandboxing](https://github.com/mozilla/addons-frontend/blob/master/tests/setup.js#L71) but I'd feel better if it were a new instance mainly as not to obscure the intention of the code 😀 

For easy access in assertions, we usually do it like:

```js
let fakeFastClick;

beforeEach(() => {
  fakeFastClick = {
    attach: sinon.stub(),
  };
});
```